polygeist/test/polygeist-opt/sycl/argpromotion.mlir

```diff
@@ -151,9 +151,9 @@ gpu.module @device_func {
   // COM: Test that the a call to a linkonce_odr function is modified.
   // COM: This function is a candidate, check that it is transformed correctly.
   func.func @callee6(%arg0: memref<?x!llvm.struct<(i32, i64)>>) attributes {llvm.linkage = #llvm.linkage<linkonce_odr>} {
-    // CHECK-LABEL: func.func @callee6
+    // CHECK-LABEL: func.func private @callee6
```

Comment:
Given that the function is now "private" in the mlir sense, do we still also need the llvm attribute "private" ? Doesn't the mlir private visibility lower to LLVM private ?